query/orders.js

```diff
@@ -0,0 +1,21 @@
+const { sql } = require('slonik')
+
+const getOrders = async (db, { username, state }) => {
+  try {
+    return await db.query(sql`
+      SELECT * 
+        FROM orders
+        WHERE 
+          user_id = (select id from users where username = ${username}) 
+          AND cart = true
+          ${state ? sql`AND state::text = ${state}` : sql``}
```

Comment:
* Quita la línea 10 ('AND cart = true'). Molpe me ha dicho que quite esa columna del esquema.
* Si me explicas lo que hace esto que has puesto, te lo doy por bueno 😉
${state ? sqlAND state::text = ${state} : sql``}